src/sfizz/Synth.cpp

```diff
@@ -577,6 +578,94 @@ int sfz::Synth::getNumCurves() const noexcept
     return numCurves;
 }
 
+std::string sfz::Synth::exportMidnam(absl::string_view model) const
+{
+    pugi::xml_document doc;
+
+    // TODO: perhaps define this somewhere, or use some metadata from the SFZ
+    const char *manufacturer = "Paul Ferrand";
+    const char *defaultModel = "Sfizz";
```

Comment:
These can maybe go in `Config.h`. Also, I think you can put `sfizz's authors` or something for manufacturer for future reference.